tests/unit/test_charm.py

```diff
@@ -649,6 +697,48 @@ def test_exec_error_on_client_deleted_event_emitted(
     assert caplog.record_tuples[0][2] == f"Exited with code: {err.exit_code}. Stderr: {err.stderr}"
 
 
+def test_config_updated_without_login_ui_endpoints_interface(
+    harness: Harness, mocked_sql_migration: MagicMock
+) -> None:
+    harness.set_can_connect(CONTAINER_NAME, True)
+    harness.charm.on.hydra_pebble_ready.emit(CONTAINER_NAME)
+    setup_postgres_relation(harness)
+
+    expected_config = {
+        "dsn": f"postgres://{DB_USERNAME}:{DB_PASSWORD}@{DB_ENDPOINT}/testing_hydra",
+        "log": {"level": "trace"},
+        "secrets": {
+            "cookie": ["my-cookie-secret"],
+            "system": ["my-system-secret"],
+        },
+        "serve": {
+            "admin": {
+                "cors": {
+                    "allowed_origins": ["*"],
+                    "enabled": True,
+                },
+            },
+            "public": {
+                "cors": {
+                    "allowed_origins": ["*"],
+                    "enabled": True,
+                },
+            },
+        },
+        "urls": {
+            "self": {
+                "issuer": "http://127.0.0.1:4444/",
+                "public": "http://127.0.0.1:4444/",
+            },
+        },
+        "webfinger": {
+            "oidc_discovery": {"supported_scope": ["openid", "profile", "email", "phone"]}
+        },
+    }
+
+    assert yaml.safe_load(harness.charm._render_conf_file()) == expected_config
```

Comment:
We could compare the expected config with the actual file that was pushed to the container instead of checking `_render_conf_file` method output.
```suggestion
    container_config = container.pull(path="/etc/config/hydra.yaml", encoding="utf-8")
    assert yaml.safe_load(container_config.read()) == expected_config
```